src/io/perun/asciidoctor.clj

```diff
@@ -0,0 +1,13 @@
+(ns io.perun.asciidoctor
+  (:require [io.perun.core :as perun]
+            [clojure.java.io :as io])
+  (:import [org.asciidoctor Asciidoctor Asciidoctor$Factory]))
+
+(defn asciidoctor-to-html [file-content]
+  (.convert (Asciidoctor$Factory/create "") file-content {}))
```

Comment:
Yes, but this fails the moment that environment-specific parameters are introduced like libraries. The issue of creating new containers came up on https://github.com/hashobject/perun/pull/100 and does hurt performance for large number of conversions.
Probably the converter-specific settings have to be raised to the task level, so that a different container setting requires a different task to be created. Then inside the task the same converter (factory) can be used.
I will refactor this.